raiden_contracts/utils/proofs.py

```diff
@@ -1,13 +1,26 @@
 from eth_abi import encode_single
 from eth_typing.evm import HexAddress
 from web3 import Web3
+from web3.types import Nonce
 
 from raiden_contracts.constants import MessageTypeId
+from raiden_contracts.utils.type_aliases import (
+    AdditionalHash,
+    BalanceHash,
+    BlockExpiration,
+    ChainID,
+    ChannelID,
+    PrivateKey,
+    Signature,
+    TokenAmount,
+)
 
 from .signature import sign
 
 
-def hash_balance_data(transferred_amount: int, locked_amount: int, locksroot: bytes) -> bytes:
+def hash_balance_data(
+    transferred_amount: int, locked_amount: int, locksroot: bytes
```

Comment:
`TokenAmount`s here as well
I also think we should have a type for the Locksroot (hash)